v1alpha1/proto/grafeas.proto

```diff
@@ -1365,3 +1410,35 @@ service Grafeas {
     };
   }
 }
+
+service GrafeasProjects {
```

Comment:
Can you add a top level service comment?

Reply:
Please have an extra look at this one. I think there is room for improvement.